realm/realm-library/src/main/java/io/realm/internal/SharedRealm.java

```diff
@@ -175,6 +192,16 @@ public boolean isInTransaction() {
         return nativeIsInTransaction(nativePtr);
     }
 
+    public void setSchemaVersion(long schemaVersion) {
+        // FIXME migrate to ObjectStore
```

Comment:
Yes, but `set_schema_version` is not declared in `object_store.hpp`.

We need to modify ObjectStore itself to implement the method on `SharedRealm`. So I'd like to leave this as it is and make another PR to do that.